mirrord/sip/src/lib.rs

```diff
@@ -0,0 +1,407 @@
+#[cfg(target_os = "macos")]
+mod codesign;
+#[cfg(target_os = "macos")]
+mod error;
+#[cfg(target_os = "macos")]
+mod whitespace;
+
+#[cfg(target_os = "macos")]
+mod main {
```

Comment:
@meowjesty when I originally created this layout, I thought maybe there's a better solution than all this `#[cfg(..)]` maybe you have a suggestion.